airflow/ui/src/components/Menu/MenuButton.tsx

```diff
@@ -34,6 +35,9 @@ const MenuButton: React.FC<Props> = ({ dag }) => (
       <ActionButton actionName="" icon={<MdMoreHoriz />} text="" />
     </Menu.Trigger>
     <Menu.Content>
+      <Menu.Item value="Run Backfill">
+        <ParseDag dagId={dag.dag_id} fileToken={dag.file_token} />
```

Comment:
If we push that to the menu I think we should make them consistent, both Item should have an Icon, or none of them.

Reply:
Okay, added an icon for Backfill... do you think this one is good?

![image](https://github.com/user-attachments/assets/ce329f6f-2a48-4b57-b712-3834578d2811)

@dstandish as "master of backfill" - is icon good in your view?

Otherwise some alternatives are: https://react-icons.github.io/react-icons/search/#q=fill

Reply:
Agreed with @dstandish that the icon is not "perfect fitting" but can be changed later anyway...